avaframe/out1Peak/outPlotAllPeak.py

```diff
@@ -164,8 +164,8 @@ def plotAllFields(avaDir, inputDir, outDir, cfg):
         nx = data.shape[1]
         Ly = ny*cellSize
         Lx = nx*cellSize
-        unit = pU.cfgPlotUtils['unit%s' % cfg['GENERAL']['peakVar']]
-
+        unit = unit
```

Comment:
What is this for? it is not needed, right?

Reply:
uh correct, removed it